.env.sample

```diff
@@ -0,0 +1,9 @@
+SECRET_KEY=nad7kycy!op3$^p@37=xk!mcwmmv5zw6@*e$r+#m5ood2)@i78
+DEBUG=on
+ENGINE=django.db.backends.postgresql
```

Comment:
Do we need this as a env var?

Reply:
No but I thought of keeping also DB variables together